docs/commands/autoview.md

```diff
@@ -0,0 +1,24 @@
+# autoview
+
+Print the content of the pipeline as a table or list.
+It is the implied or default output when none is provided.
```

Comment:
Just a couple small changes:

* You might want to mention that if you send a single values to `autoview`, it'll attempt to view it. If you send a string, and it originally comes from a source file, it'll open up `textview` so you can read it like source code. Or if it's a single value of binary, it'll try to view the contents for you.
* I think we could say "implied or default viewer" when non is provided here, since a lot of what it's doing is viewing the data you're giving it.

Reply:
Cool! Thanks for the feedback.
I've added a brief explanation about when textview and "binaryview" would be used, as well as examples for those cases too, and an extra example for a single value.
Not 100% sure about the description of the binary file case.